Common/Brokerages/IBrokerageModel.cs

```diff
@@ -146,18 +153,18 @@ public static IBrokerageModel Create(BrokerageName brokerage, AccountType accoun
 
                 case BrokerageName.TradierBrokerage:
                     return new TradierBrokerageModel(accountType);
-                    
+
                 case BrokerageName.OandaBrokerage:
                     return new OandaBrokerageModel(accountType);
-                    
+
                 case BrokerageName.FxcmBrokerage:
                     return new FxcmBrokerageModel(accountType);
 
                 case BrokerageName.Bitfinex:
                     return new DefaultBrokerageModel(accountType);
 
                 case BrokerageName.GDAX:
-                    return new GDAXBrokerageModel(accountType);
+                    return new GDAXBrokerageModel();
```

Comment:
Shouldn't we still be passing in the `accountType` here? If not, users will never get the warning if they use gdax w/ margin, and perhaps worse, if the specify gdax w/ margin it won't use margin, but would use the cash models.

Reply:
As GDAX does not currently support margin trading, we have two options:
1. silently default to `CashBuyingPowerModel`, ignoring the `accountType` argument
2. throw an exception if `AccountType.Margin` is passed

I chose option 1 only to avoid forcing people to change the argument to `AccountType.Cash` in existing algorithms.